Source/WebKit/UIProcess/ProcessThrottler.h

```diff
@@ -155,6 +161,7 @@ class ProcessThrottler : public CanMakeWeakPtr<ProcessThrottler> {
     HashSet<BackgroundActivity*> m_backgroundActivities;
     std::optional<uint64_t> m_pendingRequestToSuspendID;
     bool m_shouldTakeUIBackgroundAssertion;
+    bool m_allowActivity { true };
```

Comment:
I think this should be named `m_allowsActivities` or `m_areActivitiesAllowed`. The setter's name should be updated accordingly too.